.github/workflows/lint.yml

```diff
@@ -34,3 +33,15 @@ jobs:
       - name: Check for difference
         run: |
           git diff --exit-code
+
+  cpp-clang-format:
+    name: C++ / clang-format
+    runs-on: ubuntu-latest
+    steps:
+    - name: Initialize a repository
+      uses: actions/checkout@v4
+    - name: Run clang-format
+      uses: DoozyX/clang-format-lint-action@v0.17
+      with:
+        source: 'include lib python'
```

Comment:
I think it is better not to limit "source" here

Reply:
In this case extensions should be specified